client/src/Movies/MovieList.js

```diff
@@ -34,7 +35,7 @@ export default class MovieList extends Component {
 function MovieDetails({ movie }) {
   const { title, director, metascore, stars } = movie;
   return (
-    <div className="movie-card">
+    <div className="movie-card"><Link className='moviecard' to={`/movies/${movie.id}`}>
```

Comment:
This works fine, but the only reason for the outer div is to apply a className, you could put the className on the Link